entrypoints/docker-entrypoint.sh

```diff
@@ -6,4 +6,4 @@ set -e
 rm -f /app/tmp/pids/server.pid
 
 # Then exec the container's main process (what's set as CMD in the Dockerfile).
-exec "$@"
\ No newline at end of file
+exec "$@"
```

Comment:
probably u dont need to upload files with these type of changes like here and in Dockerfile